CHANGELOG.md

```diff
@@ -1,3 +1,95 @@
+# Release 1.17.0
+
+## Highlight
+* Enderman Slayer Helper by @akarahdev
+* fancy Party Finder GUI by @viciscat
+* dungeon puzzle Waterboard Solver by @kevinthegreat1
+* f3/m3 fire freeze staff timer by @KhafraDev
+* f3/m3 guardian health display by @KhafraDev
+* f5/m5 livid color title by @kevinthegreat1
+* remove screens when switching island by @LifeIsAParadox
+* Dungeon Score by @kevinthegreat1
```

Comment:
I would also add credit to Rime (emirlol) because they greatly enhanced the functionality of the dungeon score.